.github/workflows/build-containers.yml

```
@@ -20,7 +20,7 @@ on:
     types: [published]
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.run_number }}
+  group: build_containers--${{github.ref}}-${{github.event.pull_request.number || github.run_number}}
```

(typo) Can be done in a following PR too:
```suggestion
  group: build_containers-${{github.ref}}-${{github.event.pull_request.number || github.run_number}}
```